packages/cf-component-select/src/Select.js

```diff
@@ -76,7 +148,143 @@ Select.propTypes = {
   multi: PropTypes.bool,
   onChange: PropTypes.func,
   async: PropTypes.bool,
-  creatable: PropTypes.bool
+  creatable: PropTypes.bool,
+  menuRenderer: PropTypes.func,
+  arrowRenderer: PropTypes.func,
+  clearRenderer: PropTypes.func,
+  isValidNewOption: PropTypes.func,
+  noResultsText: PropTypes.string,
+  promptTextCreator: PropTypes.func,
+  newOptionCreator: PropTypes.func,
+  menuTitle: PropTypes.string
+};
+
+const singleOrPlaceholder = theme => ({
+  bottom: 0,
+  color: theme.colorGrayDark,
+  left: 0,
+  lineHeight: '32px',
+  paddingLeft: '10px',
+  paddingRight: '10px',
+  position: 'absolute',
+  right: 0,
+  top: 0,
+  maxWidth: '100%',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap'
+});
+
+const styles = ({ theme, disabled }) => {
+  const values = {
+    boxSizing: 'border-box',
+    fontSize: '0.86667rem'
+  };
+
+  return {
+    ...values,
+    '& div': values,
+    '& input': values,
+    '& span': values,
+
+    flex: theme.flex,
+    margin: theme.margin,
+    width: theme.width,
+    display: theme.display,
+    verticalAlign: theme.verticalAlign,
+    position: 'relative',
+    opacity: disabled ? 0.5 : 1,
+
+    '& .Select-control': {
+      backgroundColor: disabled ? theme.colorGrayDark : theme.colorWhite,
+      border: `1px solid ${theme.color.hail}`,
+      borderRadius: theme.borderRadius,
+      color: theme.colorGrayDark,
+      cursor: 'default',
+      display: 'table',
+      borderSpacing: 0,
+      borderCollapse: 'separate',
+      height: '32px',
+      outline: 'none',
+      overflow: 'hidden',
+      position: 'relative',
+      width: '100%',
+      transition: 'border-color 0.2s ease'
+    },
+    '& .Select-control .Select-input:focus': {
+      outline: 'none'
+    },
+    '& .Select-menu-outer': {
+      position: 'relative'
+    },
+    '&:hover .Select-control': {
+      borderColor: theme.colorBlueDark
```

Comment:
I'm going to add the new hover colors today. So we can swap these out.

Reply:
Can you add black and white as well?